src/test/org/codehaus/groovy/classgen/asm/sc/bugs/Groovy9892Bug.groovy

```diff
@@ -0,0 +1,21 @@
+package org.codehaus.groovy.classgen.asm.sc.bugs
```

Comment:
license header is expected here:
https://github.com/apache/groovy/blob/494c7ec11799c7b6c91f6d7689470544800b7416/src/main/java/org/codehaus/groovy/classgen/asm/sc/StaticTypesCallSiteWriter.java#L1-L18

Reply:
License header is now added